package.json

```diff
@@ -45,6 +45,7 @@
     "express-prom-bundle": "^2.1.0",
     "express-session": "^1.14.1",
     "express-winston": "^2.0.0",
+    "fetch-jsonp": "^1.0.6",
```

Comment:
MailChimp does not support sending data cross-origin, but they do support JSONP, which is not supported by pure `fetch`.

Therefore `fetch-jsonp` to the rescue (to avoid building a proxy API for a temporary beta feature).